benchto-driver/src/main/java/com/teradata/benchto/driver/DriverApp.java

```diff
@@ -74,6 +75,7 @@ public static void main(String[] args)
 
         try {
             executionDriver.execute();
+            SECONDS.sleep(10); // Give time for TaskExecutor to run callbacks
```

Comment:
You can't wait for termination without shutting down first. This would work iff no listener uses `TaskExecutor` under the hood. + this is not nice to shutdown arbitrary bean before context gets destroyed.